containers/sprm-to-json/context/main.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+from glob import glob
+from pathlib import Path
+from os import makedirs
+import json
+
+from shapely.geometry import Polygon
+import pandas as pd
+import numpy as np
+
+# Number of vertices for the polygon boundary data.
+NUM_VERTICES = 20
+GENES_FILE_SUFFIX = ".ome.tiff-cell_channel_mean.csv"
+CLUSTER_FILE_SUFFIX = ".ome.tiff-cell_cluster.csv"
+POLYGON_FILE_SUFFUX = ".ome.tiff-cell_polygons_spatial.csv"
+
+
+def sprm_to_items(input_file):
+    df = pd.read_csv(input_file)
+    df = df.set_index("ID")
+    df_items = df.T.to_dict().items()
+    return df_items
+
+
+def write_genes_or_factors_to_cells(input_file, cells={}, genes=True):
```

Comment:
if this took `df_items` instead of `input_file` you could give a little doctest to explain what it does.

Reply:
I'll add an issue for doctest since we don't do it for anything else here and we already have tests.  I am not familiar with setting that up.